diff -aurN leafpad-0.8.17.orig/data/leafpad.desktop.in leafpad-0.8.17/data/leafpad.desktop.in
--- leafpad-0.8.17.orig/data/leafpad.desktop.in	2009-12-31 13:29:29.000000000 +0100
+++ leafpad-0.8.17/data/leafpad.desktop.in	2009-12-31 13:29:38.000000000 +0100
@@ -6,5 +6,5 @@
 Icon=leafpad
 Terminal=false
 Type=Application
-MimeType=text/plain
+MimeType=text/plain;
 Categories=GTK;Utility;TextEditor;
